### PR TITLE
Feat/update room

### LINKED ISSUE #116: Update room

### PR DESCRIPTION
**Pull request by vzart**

### CODE REVIEW

File: src/Application/Rooms/Commands/UpdateRoom.cs [hunk 1]
```diff
@@ -8,8 +13,67 @@ public class UpdateRoom
...
+            }
+            
+            var nameExisted = await _context.Rooms.AnyAsync(x => x.Name
+                .ToLower().Equals(request.Name.ToLower())
+                && x.Id != room.Id
+                , cancellationToken: cancellationToken);
+
+            if (nameExisted)
+            {
+                throw new ConflictException("New name has already exists.");
```
Comment (ChienNQuang) [inline src/Application/Rooms/Commands/UpdateRoom.cs]: english

Comment (vzart) [reply src/Application/Rooms/Commands/UpdateRoom.cs]: your test, your english man

Comment (ChienNQuang) [reply src/Application/Rooms/Commands/UpdateRoom.cs]: and you blindly follow a mistake?

File: src/Application/Rooms/Commands/UpdateRoom.cs [hunk 2]
```diff
@@ -1,15 +1,99 @@
...
 {
+    public class Validator : AbstractValidator<Command>
+    {
+        public Validator()
+        {
+            RuleLevelCascadeMode = CascadeMode.Stop;
+
+            RuleFor(x => x.Name)
+                .NotEmpty().WithMessage("Name can not be empty.");
+
```
Comment (kaitoz11) [inline src/Application/Rooms/Commands/UpdateRoom.cs]:
```C#
RuleFor(x => x.Name)
    .NotEmpty().WithMessage("Name can not be empty.");
    .MaximumLength(64).WithMessage("Name can not exceed 64 characters.");
```

**Review by kaitoz11 — APPROVED** [review]
LGTM

**Pull request merged**